packages/ember-routing-htmlbars/tests/helpers/closure_action_test.js

```diff
@@ -583,3 +584,38 @@ QUnit.test('objects that define INVOKE can be casted to actions', function(asser
 });
 
 }
+
+if (isEnabled('ember-improved-instrumentation')) {
+  QUnit.test('action should fire interaction event', function(assert) {
+    assert.expect(2);
+
+    subscribe('interaction.ember-action', {
+      before() {
+        console.log(arguments);
+        assert.ok(true, 'instrumentation subscriber was called');
+      },
+      after() {
+      }
+    });
+
+    innerComponent = EmberComponent.extend({
+      fireAction() {
+        this.attrs.submit();
+      }
+    }).create();
+
+    outerComponent = EmberComponent.extend({
+      layout: compile('{{view innerComponent submit=(action outerSubmit)}}'),
```

Comment:
I'm assuming the right thing to use would be a component helper, though I'm unsure the best way to do a component setup in this test. Should I try to use [RenderingTest](https://github.com/emberjs/ember.js/blob/master/packages/ember-glimmer/tests/integration/components/dynamic-components-test.js#L7)? Or perhaps do some custom setup like in [local-lookup-test.js](https://github.com/emberjs/ember.js/blob/master/packages/ember-htmlbars/tests/integration/local-lookup-test.js)?